core/chain-configs/src/test_utils.rs

```diff
@@ -371,6 +371,7 @@ impl ClientConfig {
             save_invalid_witnesses: false,
             transaction_request_handler_threads: default_rpc_handler_thread_count(),
             protocol_version_check: Default::default(),
+            enable_early_prepare_transactions: cfg!(feature = "nightly"), // Allows testing enabled and disabled in CI
```

Comment:
That's atypical, usually you'd implement tests for both enabled and disabled. More importantly once this feature is stabilized we should always be testing it in regular non-nightly runs.